### ISSUE TITLE
Not Working

### ISSUE THREAD

**Issue by wamobileapps** [przelewy24/p24-mobile-lib-ios#2]
I integrated przelewy24 library in my iOS app, But I try direct method is not calling . Here is my code

func PaymentMethod() {
        
      
        
        let transactionParams = P24TransactionParams();
        transactionParams.merchantId = Int32(objTeacherAvailability.getmerchantId());
        transactionParams.crc = objTeacherAvailability.getcrc();
        transactionParams.sessionId = objTeacherAvailability.getsessionId();
        transactionParams.address = "Test street";
        transactionParams.amount = 400;
        transactionParams.city = "Poznan";
        transactionParams.zip = "61-600";
        transactionParams.client = "John smith";
        transactionParams.country = "PL";
        transactionParams.language = "pl";
        transactionParams.currency = "PLN";
        transactionParams.email = "test@test.pl";
        transactionParams.phone = "1223134134";
        transactionParams.desc = "test payment description";
      
       
       let params = P24TrnDirectParams.init(transactionParams: transactionParams)!
        params.sandbox = true
       P24.startTrnDirect(params, in: self, delegate: self)
    }

I am getting this 

https://ibb.co/g4ZRV0

Anybody let me know what is wrong here ?

**Comment by mariachi89** [comment]
Sounds like issue in newest library version.

I've got the same issue when application runs directly from xcode.
When application is built to .ipa file or uploaded to TestFlight, then I'm getting a red information says that checksum is wrong.

All transaction parameters sent to the iOS library were tested with Android library, but issue does not exist there.

I've reported the issue to P24 helpline - waiting to response.
